### PR TITLE
SimpleInterest

### PR DESCRIPTION
**Pull request by april-EXO**
[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### Describe your change:

- [ x] Add an algorithm?
- [ ] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

### Checklist:

- [ x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
- [ x] This pull request is all my own work -- I have not plagiarized.
- [ x] I know that pull requests will not be merged if they fail the automated tests.
- [ x] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [ x] All new JavaScript files are placed inside an existing directory.
- [ x] All filenames should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
- [ x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [ x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: Maths/SimpleInterest.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/**
+ * @function SimpleInterest
+ * @description to calculate the amount of interest charged on a sum at a given rate and for a given period of time
+ * @param {number} principal - input: the principal amount
+ * @param {number} intrestRate - input: the rate of interest in percentage
```
Comment (appgurueu) [inline Maths/SimpleInterest.js:5]: interest*

File: Maths/SimpleInterest.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+ * @param {number} time - input: the time period in year
+ * @return {number} simpleInterest
+ */
+
+const SimpleInterest = (principal, interestRate, time) => {
+  const simpleInterest = parseInt((principal * interestRate * time) / 100)
```
Comment (appgurueu) [inline Maths/SimpleInterest.js:11]: Use `Math.floor` instead of abusing `parseInt` to floor a float.

**Review by appgurueu — COMMENTED** [review]
Might be too trivial to warrant inclusion

**Review by raklaptudirm — COMMENTED** [review]
I would say this algorithm does not showcase any important aspects of JavaScript, nor is it an interesting or popular algorithm. Closing this pull request as the algorithm is too trivial (2/2).

**Pull request closed without merging**